pkg/domain/infra/abi/play.go

```diff
@@ -991,7 +992,11 @@ func (ic *ContainerEngine) getImageAndLabelInfo(ctx context.Context, cwd string,
 		buildOpts := new(buildahDefine.BuildOptions)
 		commonOpts := new(buildahDefine.CommonBuildOptions)
 		buildOpts.ConfigureNetwork = buildahDefine.NetworkDefault
-		buildOpts.Isolation = buildahDefine.IsolationChroot
+		isolation, err := bparse.IsolationOption("")
```

Comment:
This grabs the environment variable setting, right? If so, maybe add a comment to that effect, I'd be tempted to change this to a constant otherwise.

Reply:
It will use the buildah default or if environment variable is set, will use this.  This could be considered a breaking change, although I think is is a security issue.  We should be using the tightest security possible.  If users need to use chroot, they can just set the environment variable back.